### ISSUE TITLE
Find PDF with drawing of the subcomponents

### ISSUE THREAD

**Issue by christoph-maurer** [building-envelope-data/database#47]
On https://www.solarbuildingenvelopes.com/data, I see the data set
| 58c25ae9-1577-41c3-b113-03cb749a77c9 | BIPV Glazing Example, technical drawing, PDF | 34f7aebd-5d11-4db7-a5d0-18b51af8f817 | Locator https://www.solarbuildingenvelopes.com/api/resources/e534de7a-09a3-47d7-a77d-b58815dd5bab | Data Format UUID e8cd8dce-4e1f-4219-b9cd-74a0ae55e98c |
|---|---|---|---|---|

However, when I query https://www.buildingenvelopedata.org/data or https://www.solarbuildingenvelopes.com/data or https://www.solarbuildingenvelopes.com/data/optical with "Data Format ID is equal to e8cd8dce-4e1f-4219-b9cd-74a0ae55e98c" , I get no results. @simon-wacker Do you know why?

**Comment by simonwacker** [comment]
Yes, it is not optical data but photovoltaic data. That's why it doesn't even show on the optical data page without any filters. Filtering for the mentioned data format on https://www.solarbuildingenvelopes.com/data/photovoltaic yields the entry.

**Comment by christoph-maurer** [comment]
Thank you! Do you know why it doesn't work on https://www.solarbuildingenvelopes.com/data ?
